website/project/model.py

```diff
@@ -2196,6 +2195,8 @@ def fork_node(self, auth, title='Fork of '):
             save=False
         )
 
+        forked.save()
```

Comment:
for clarification, is the `save` here so we get a guid assigned before calling `add_log` and creating external references?